test/parallel_api/algorithm/alg.sorting/partial_sort.pass.cpp

```diff
@@ -55,23 +55,79 @@ struct Num
 #endif
 
 template <typename Type>
-struct test_brick_partial_sort
+struct test_without_compare
```

Comment:
Will it help to reduce the number of code lines if we combine this test with the test below?

Reply:
We can call only one algorithm in a single `invoke_on_all_policies` for having unique kernel names, so we cannot fuse the cases to remove many lines. 
We can leave one structure (e.g. `test_brick_partial_sort`) and add more overloads of `operator(...)`. This will reduce the test (~4 lines).

Reply:
Though we can withdraw data generation to a common place.